### PR TITLE
Css

### LINKED ISSUE #63: Website flow
As a user, I want the website to be smooth and easy to navigate.
Link all the HTML pages.

### LINKED ISSUE #66: Add header/footer
EJS partials are created but are empty, need to have header and footer for every page

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/views/partials/header.ejs [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+
+    <link href="css/index.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <header>
+      <a href="/profile"><h2 class="userProfile">User Profile</h2></a>
+      <a href="/signin"><h2 class="signUp">Login</h2></a>
+      <h1><a href="/">Let's Eat</a></h1>
+    </header>
+  </body>
```
Comment (aaahmed1) [inline src/views/partials/header.ejs:20]: Remove this.

File: src/views/partials/header.ejs [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+</html>
```
Comment (aaahmed1) [inline src/views/partials/header.ejs]: Remove the `</html>` and `</body>` tags

File: src/views/partials/footer.ejs [hunk 3]
```diff
@@ -1,7 +1,27 @@
-</main>
-<!--Todo: Footer-->
-<% if (typeof libs !== 'undefined') { %>
+</footer>
```
Comment (aaahmed1) [inline src/views/partials/footer.ejs]: Change this to `<footer>`

File: src/views/partials/header.ejs (hunk 2 above)
Comment (ghost) [reply src/views/partials/header.ejs]: I need to put those tags.

File: src/views/partials/header.ejs [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
-    <% } %>
-<% } %>
-</head>
-<body>
-  <!--Todo: Navbar-->
-  <main>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <!-- <meta http-equiv="X-UA-Compatible" content="IE=edge" /> -->
```
Comment (JacobHarland) [inline src/views/partials/header.ejs]: Why are these 2 boilerplates commented out? Why are the if statements on line 8-10 commented out? The first if command should have index.css and then you can remove the style from the controllers I believe

File: src/views/partials/header.ejs [hunk 5]
```diff
@@ -1,16 +1,19 @@
...
-  <title><%- title %></title>
-  <% if (typeof styles !== 'undefined') { %>
-    <% if (styles.length > 0) { %>
-            <link rel="stylesheet" href="css/<%= styles %>.css">
-    <% } %>
-<% } %>
-</head>
-<body>
-  <!--Todo: Navbar-->
-  <main>
```
Comment (JacobHarland) [inline src/views/partials/header.ejs:16]: Main shouldn't be removed

File: src/views/partials/footer.ejs [hunk 6]
```diff
@@ -1,7 +1,27 @@
-</main>
```
Comment (JacobHarland) [inline src/views/partials/footer.ejs:1]: Main shouldn't be removed

File: src/views/partials/footer.ejs [hunk 7]
```diff
@@ -1,7 +1,27 @@
...
-<!--Todo: Footer-->
-<% if (typeof libs !== 'undefined') { %>
+<footer>
+<!-- <% if (typeof libs !== 'undefined') { %>
```
Comment (JacobHarland) [inline src/views/partials/footer.ejs]: This needs to be uncommented, the survey page JS doesn't work without it

File: src/controllers/testController.ts [hunk 8]
```diff
@@ -31,3 +31,7 @@ export const survey = (req: Request, res: Response) => {
 export const thankyou = (req: Request, res: Response) => {
   res.render("thankyou", { styles: "index", title: "Thank You!" });
 };
+
+export const about = (req: Request, res: Response) => {
+  res.render("about", { styles: "about", title: "About" });
```
Comment (JacobHarland) [inline src/controllers/testController.ts]: passing in about styles but no about.css exists yet

**Review by JacobHarland — CHANGES_REQUESTED** [review]
Multi comment review

File: src/views/partials/footer.ejs [hunk 9]
```diff
@@ -1,7 +1,25 @@
-</main>
-<!--Todo: Footer-->
+<footer>
 <% if (typeof libs !== 'undefined') { %>
```
Comment (JacobHarland) [inline src/views/partials/footer.ejs]: Script tags need to be right above body, so just cut paste this right before the closing body tag

File: src/views/partials/header.ejs [hunk 10]
```diff
@@ -1,16 +1,23 @@
...
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title><%- title %></title>
+    <!-- <% if (typeof styles !== 'undefined') { %> <% if (styles.length > 0) { %>
+    <link rel="stylesheet" href="css/<%= styles %>.css" />
```
Comment (JacobHarland) [inline src/views/partials/header.ejs:9]: Can leave the if statement commented, but need to uncomment the styles.css line, this is how the page knows which specific css file needs to be loaded

File: src/views/partials/header.ejs [hunk 11]
```diff
@@ -1,16 +1,23 @@
...
+  </head>
+  <body>
+    <main>
+    <header>
+      <a href="/profile"><h2 class="userProfile">User Profile</h2></a>
+      <a href="/signin"><h2 class="signUp">Login</h2></a>
+      <a href="/"><h1>Let's Eat</h1></a>
+    </header>
+    
+  </body>
```
Comment (JacobHarland) [inline src/views/partials/header.ejs]: Delete the closing body and html tag, they are closed in the footer partial

**Review by JacobHarland — CHANGES_REQUESTED** [review]

File: src/views/partials/header.ejs (hunk 10 above)
Comment (ghost) [reply src/views/partials/header.ejs:9]: we only have one css file. look at the line below it

**Review by JacobHarland — APPROVED** [review]

**Pull request merged**